lib/model/autocomplete.dart

```diff
@@ -350,9 +406,19 @@ class AutocompleteDataCache {
   void invalidateUser(int userId) {
     _nameWordsByUser.remove(userId);
   }
+
+  List<String> nameWordsForTopic(Topic topic) {
+    return _nameWordsByTopic[topic.value] ??= topic.value.toLowerCase().split(' ');
+  }
+
+  void invalidateTopic(String value) {
+    _nameWordsByTopic.remove(value);
+  }
```

Comment:
Bump on https://github.com/zulip/zulip-flutter/pull/627#discussion_r1663327014 ; "name words" is confusing when talking about topics.